app/containers/message/Message.tsx

```diff
@@ -83,13 +83,55 @@ const MessageInner = React.memo((props: IMessageInner) => {
 });
 MessageInner.displayName = 'MessageInner';
 
-const Message = React.memo((props: IMessage) => {
+const Message = React.memo((props: IMessageTouchable & IMessage) => {
+	const handleMentionsOnAccessibilityLabel = (label: string) => {
+		const { mentions = [], channels = [] } = props;
+
+		mentions.forEach(item => {
+			if (item?.username) {
+				label = label.replaceAll(`@${item.username}`, item.username);
+			}
+		});
+
+		channels.forEach(item => {
+			if (item?.name) {
+				label = label.replaceAll(`#${item.name}`, item.name);
+			}
+		});
+
+		return label;
+	};
+
+	// temp accessibilityLabel
+	const accessibilityLabel = useMemo(() => {
+		let label = '';
+		label = props.isInfo ? (props.msg as string) : `${props.tmid ? `thread message ${props.msg}` : props.msg}`;
+		if (props.isThreadReply) {
+			label = `replying to ${props.tmid ? `thread message ${props.msg}` : props}`;
+		}
+		if (props.isThreadSequential) {
+			label = `thread message ${props.msg}`;
+		}
+		if (props.isEncrypted) {
+			label = i18n.t('Encrypted_message');
+		}
+		if (props.isInfo) {
+			// @ts-ignore
+			label = getInfoMessage({ ...props });
+		}
+		label = handleMentionsOnAccessibilityLabel(label);
+
+		const hour = props.ts ? new Date(props.ts).toLocaleTimeString() : '';
+		const user = props.useRealName ? props.author?.name : props.author?.username || '';
+		return `${user} ${hour} ${label}`;
+	}, []);
```

Comment:
Missing lots of dependencies here.
After a message is rendered, then edited, this accessibility label might keep the previous value.